static/js/popovers.js

```diff
@@ -842,12 +842,12 @@ export function register_click_handlers() {
         const row = $(this).closest(".message_row");
         e.stopPropagation();
         const message = message_lists.current.get(rows.id(row));
-        const group = user_groups.get_user_group_from_id(user_group_id, true);
-        if (group === undefined) {
+        try {
+            const group = user_groups.get_user_group_from_id(user_group_id, true);
```

Comment:
The `true` argument should be removed (same issue).

Reply:
Fixed both of these.